node/Cargo.toml

```diff
@@ -1,18 +1,13 @@
 [package]
 name = "metaquity-network"
-# SBP-M1 review: consider resetting version as new project
-version = "4.0.0-dev"
-# SBP-M1 review: update description
-description = "A fresh FRAME-based Substrate node, ready for hacking."
-# SBP-M1 review: add metaquity as author
-authors = ["Substrate DevHub <https://github.com/substrate-developer-hub>"]
-# SBP-M1 review: set homepage
-homepage = "https://substrate.io/"
-edition = "2021"
+version = "0.1.0"
+description = "Metaquity Network is the world's first permissioned blockchain and DeFi protocol for Alternative Real World Assets (ARWA)."
```

Comment:
This can be `Metaquity Network is the world's first permission blockchain and DeFi protocol for Real World Assets (ARWA).` @abhichat85 @ArkoGanguli what do you think?

Reply:
Done